onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -51,12 +51,19 @@ namespace perftest {
       "\t-d [cudnn_conv_algorithm]: Specify CUDNN convolution algothrithms: 0(benchmark), 1(heuristic), 2(default). \n"
       "\t-q: [CUDA only] use separate stream for copy. \n"
       "\t-z: Set denormal as zero. When turning on this option reduces latency dramatically, a model may have denormals.\n"
+      "\t-i: [OpenVINO only] Specify OpenVINO runtime options as key value pairs. Different options available are: \n"
```

Comment:
I was thinking -i could be used for any EP.  wonder if there's a way to convey that it's generic,
while at the same time documenting the possible options for OpenVINO EP

Reply:
Yeah, I made -i flag EP generic now. It can be used by any EP's to pass their respective run time options as key-value pairs.
Here's how it looks now.
![image](https://user-images.githubusercontent.com/29036417/107852393-7b3fcc00-6e36-11eb-9682-1b8dca5ac9e4.png)